server/database/schemas/users.py

```diff
@@ -0,0 +1,24 @@
+from pydantic import BaseModel
+from uuid import UUID
+
+
+class UserBase(BaseModel):
+    """Common user data shared by all schemas"""
+
+    email: str
+    settings: str
```

Comment:
Wouldn't it have been better to able set the `settings` parameter to be None by default so it would be easier to apply default settings while needed?

And why should it be a string? If it is a string, then it would be very useful to write some docs about values, which could be written here

Reply:
It's implied that the settings string is a json config storing the settings variables. Having mixed column types is potentially complicated, I think it would be better to store an empty '{}' json string by default instead, but that's up for debate.

Reply:
Fixed